priv/app/function_browser.jsx

```diff
@@ -107,7 +105,7 @@ export default class FunctionBrowser extends React.Component {
   }
 
   matchFunSignature(input) {
-    var regex = /(\w+):(\w+)\/(\d+)/;
+    var regex = /([\w\.]+):(\w+)\/(\d+)/;
```

Comment:
I'd ignore the other characters until somebody raises an issue. I don't know much about elixir,  but in the erlang world they aren't used frequently